modules/ROOT/pages/test-recorder-examples.adoc

```diff
@@ -0,0 +1,66 @@
+= Recording a Test from an Exchange Example
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-partial:
+
+The following example shows you how to import to Studio an existing project from Exchange, and then record the flow processing to create a unit test for the flows.
```

Comment:
Just because it is the first mention in a Level-1 topic....
```suggestion
The following example shows you how to import to Anypoint Studio an existing project from Anypoint Exchange and then record the flow processing to create a unit test for the flows.
```